fixed-hash/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fixed-hash"
-version = "0.5.2"
+version = "0.5.3"
```

Comment:
I'm afraid removing libc was a breaking change for fixed-hash and all of its reverse dependencies.